src/pegout/components/FlyoverRbtcInputAmount.vue

```diff
@@ -98,23 +105,46 @@ export default defineComponent({
   setup(_, context) {
     const environmentContext = EnvironmentContextProviderService.getEnvironmentContext();
     const focus = ref(false);
-    const isMaxValueZero = ref(false);
     const rbtcAmount = ref('');
     const amountStyle = ref('');
     const stepState = ref<'unset' | 'valid' |'error'>('unset');
-
     const web3SessionState = useState<SessionState>('web3Session');
-    const minMaxValues = useGetter<{minValue: WeiBig; maxValue: WeiBig}>('flyoverPegout', constants.FLYOVER_PEGOUT_GET_MIN_MAX_VALUES);
     const setRbtcAmount = useAction('flyoverPegout', constants.FLYOVER_PEGOUT_ADD_AMOUNT);
     const addAmount = useAction('pegOutTx', constants.PEGOUT_TX_ADD_AMOUNT);
     const calculateFee = useAction('pegOutTx', constants.PEGOUT_TX_CALCULATE_FEE);
-    const account = computed<string>(() => web3SessionState.value.account as string);
     const estimatedBtcToReceive = useGetter<SatoshiBig>('pegOutTx', constants.PEGOUT_TX_GET_ESTIMATED_BTC_TO_RECEIVE);
+    const getRbtcGasFee = useGetter<Promise<WeiBig>>('web3Session', constants.SESSION_GET_RBTC_GAS_FEE);
+    const pegoutConfiguration = useStateAttribute<PegoutConfiguration>('pegOutTx', 'pegoutConfiguration');
+
+    const isValidAmount = (amount: WeiBig) => {
+      const { minValue, maxValue } = pegoutConfiguration.value;
+      const { balance } = web3SessionState.value;
+      return isRBTCAmountValidRegex(amount.toRBTCString())
+          && amount.gte(minValue)
+          && amount.lte(balance)
+          && amount.lte(maxValue);
+    };
+
+    const rbtcAmountModel = computed({
+      get() {
+        return rbtcAmount.value;
+      },
+      set(amount: string) {
+        rbtcAmount.value = amount;
+        const weiBigAmount = new WeiBig(amount, 'rbtc');
+        if (isValidAmount(weiBigAmount)) {
+          setRbtcAmount(weiBigAmount);
+          addAmount(weiBigAmount)
+            .then(() => calculateFee());
+          context.emit('get-quotes');
+        }
+      },
+    });
 
     const safeAmount = computed((): WeiBig => new WeiBig(rbtcAmount.value ?? '0', 'rbtc'));
 
     const amountErrorMessage = computed(() => {
-      const { minValue, maxValue } = minMaxValues.value;
+      const { minValue, maxValue } = pegoutConfiguration.value;
```

Comment:
Please set default values inside the peginConfiguration class: min 0.005 and max 1

Reply:
You mean this value (https://github.com/rsksmart/2wp-app/blob/feature/pegout-flow-2/src/main.ts#L20)?